src/OSPSuite.R/Domain/SensitivityAnalysis.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using OSPSuite.Core.Domain;
+
+namespace OSPSuite.R.Domain
+{
+   public class SensitivityAnalysis
```

Comment:
This will be the object used from R. Ref to simulation and list of parmaeters to vary